docs/usage.md

```diff
@@ -12,57 +12,76 @@ You will need to create a samplesheet with information about the samples you wou
 
-```
+Further information about the command line arguments is documented on the [nf-core website](https://nf-co.re/mhcquant/dev/parameters) or by using `--help`.
 
-| Column              | Description                                                                                                                                                          |
-| ------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `ID`                | An incrementing value which acts as a unique number for the given sample                                                                                             |
-| `Sample`            | Custom sample name. This entry will be identical for multiple MS runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`). |
-| `Condition`         | Additional information of the sample can be defined here.                                                                                                            |
-| `ReplicateFileName` | Full path to the MS outcome file. These files have the extentions ".raw", ".mzML" or ".d"                                                                            |
+## Rescoring using MS²Rescore
 
-An [example samplesheet](../assets/samplesheet.tsv) has been provided with the pipeline.
+By default the pipline generates additional features using MS²PIP and DeepLC via the MS²Rescore framework (`--feature_generators deeplc,ms2pip`). Additional feature generators can be added (`basic,deeplc,ms2pip`) to boost identification rates and quality. Please make sure you provide the correct `--ms2pip_model` (default: `Immuno-HCD`). All available MS²PIP models can be found on [GitHub](https://github.com/compomics/ms2pip).
+
+MS²Rescore creates a comprehensive QC report of the added features used for rescoring. This report is only available if `--rescoring_engine mokapot` is specified (default: `percolator`). The report can be found in `<OUTDIR>/multiqc/ms2rescore`. Further information on the tool itself can be read up in the published paper [Declerq et al. 2022](<https://www.mcponline.org/article/S1535-9476(22)00074-3/fulltext>)
 
 ## Running the pipeline
 
 The typical command for running the pipeline is as follows:
 
 ```console
-nextflow run nf-core/mhcquant --input 'samples.tsv' --outdir <OUTDIR> --fasta 'SWISSPROT_2020.fasta' --use_deeplc --use_ms2pip -profile docker
+nextflow run nf-core/mhcquant \
+  --input 'samplesheet.tsv' \
+  --outdir <OUTDIR> \
+  --fasta 'SWISSPROT_2020.fasta' \
+  <search parameters> \
```

Comment:
Instead of search parameters, maybe use some parameters for a class I search that we use most often?